apps/credential-service/lib/server.ts

```diff
@@ -112,11 +112,14 @@ async function wsHandler(ws: WebSocket, req: IncomingMessage) {
  */
 async function upgradeHandler(req: IncomingMessage, socket: Duplex, head: Buffer) {
   try {
-    if (!authorizationRegex.test(req.headers.authorization!)) {
+    if (!authorizationRegex.test(req.headers.authorization || '')) {
       throw Error('Missing or invalid authorization header syntax');
     }
-    /** The NgDev token from the user to be verified. */
-    const [_, ngDevToken] = req.headers.authorization!.match(/Bearer (.*)/)!;
+    /**
+     * The NgDev token from the user to be verified.
+     * We use a non-null assertion as the regex was already tested above.
+     */
+    const [_, ngDevToken] = authorizationRegex.exec(req.headers.authorization!)!;
```

Comment:
just FYI the whole non-nullability could be trivially inferred by TypeScript if you made an explicit check above

Reply:
Also lint is failing